config/initializers/time.rb

```diff
@@ -1,7 +1,8 @@
 # Extends time class with convenience methods
 class Time
   def display_date
-    getlocal.strftime("%Y-%m-%d")
+    # not being converted to /
+    getlocal.strftime("%m/%d/%Y")
```

Comment:
Looks good to me -- 
![screen shot 2017-08-13 at 9 06 22 pm](https://user-images.githubusercontent.com/3866868/29255018-954182ae-806b-11e7-983f-9f5bef125786.png)

Can probably kill that comment, it works as far as I can tell. I'd give your rails server a restart just in case and make sure though.